package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "to-do-list",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>\r <div align=\"center\">",
   "private": "true",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
- [ ] Please install and use Jest for the testing, like you have done it for the add and delete item functionality tests